### PR TITLE
brings back limited ammo for secborgs

### PR DESCRIPTION
**Pull request by ShadowDeath6**
https://github.com/yogstation13/yogstation/issues/833
disable my ass daddy

:cl: ShadowDeath6
rscadd: Due to loud outcry from Grayshirts Organized, NanoTrasen has reverted the upgrade providing their security cyborgs with unlimited disabler beams.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/energy/stun.dm [hunk 1]
```diff
@@ -31,11 +31,27 @@
 	desc = "An integrated hybrid taser that draws directly from a cyborg's power cell. The weapon contains a limiter to prevent the cyborg's power cell from overheating."
 	can_flashlight = 0
 	can_charge = 0
+	var/recharge_time = 10
 
-/obj/item/weapon/gun/energy/gun/advtaser/cyborg/newshot()
+/obj/item/weapon/gun/energy/gun/advtaser/cyborg/New()
 	..()
+	SSobj.processing |= src
```
Comment (X-TheDark) [inline code/modules/projectiles/guns/energy/stun.dm]: START_PROCESSING(SSobj, src)

File: code/modules/projectiles/guns/energy/stun.dm [hunk 2]
```diff
@@ -31,11 +31,27 @@
...
+	var/recharge_time = 10
 
-/obj/item/weapon/gun/energy/gun/advtaser/cyborg/newshot()
+/obj/item/weapon/gun/energy/gun/advtaser/cyborg/New()
 	..()
+	SSobj.processing |= src
+
+
+/obj/item/weapon/gun/energy/gun/advtaser/cyborg/Destroy()
+	SSobj.processing.Remove(src)
```
Comment (X-TheDark) [inline code/modules/projectiles/guns/energy/stun.dm]: STOP_PROCESSING(SSobj, src)

File: code/modules/projectiles/guns/energy/stun.dm (hunk 1 above)
Comment (X-TheDark) [inline code/modules/projectiles/guns/energy/stun.dm]: START_PROCESSING(SSobj, src)

File: code/modules/projectiles/guns/energy/stun.dm (hunk 2 above)
Comment (X-TheDark) [inline code/modules/projectiles/guns/energy/stun.dm]: STOP_PROCESSING(SSobj, src)

File: code/modules/projectiles/guns/energy/stun.dm [hunk 3]
```diff
@@ -48,7 +64,25 @@
 	name = "cyborg disabler"
 	desc = "An integrated disabler that draws from a cyborg's power cell. This weapon contains a limiter to prevent the cyborg's power cell from overheating."
 	can_charge = 0
+	var/recharge_time = 4
+
 
-/obj/item/weapon/gun/energy/disabler/cyborg/newshot()
+/obj/item/weapon/gun/energy/disabler/cyborg/New()
 	..()
+	SSobj.processing |= src
```
Comment (X-TheDark) [inline code/modules/projectiles/guns/energy/stun.dm]: START_PROCESSING(SSobj, src)

File: code/modules/projectiles/guns/energy/stun.dm [hunk 4]
```diff
@@ -48,7 +64,25 @@
...
+
 
-/obj/item/weapon/gun/energy/disabler/cyborg/newshot()
+/obj/item/weapon/gun/energy/disabler/cyborg/New()
 	..()
+	SSobj.processing |= src
+
+
+/obj/item/weapon/gun/energy/disabler/cyborg/Destroy()
+	SSobj.processing.Remove(src)
```
Comment (X-TheDark) [inline code/modules/projectiles/guns/energy/stun.dm]: STOP_PROCESSING(SSobj, src)

**Review by X-TheDark — CHANGES_REQUESTED** [review]
Change this

**Comment by ShadowDeath6** [comment]
I don't know how to fix reviews

**Review by X-TheDark — APPROVED** [review]

**Review by X-TheDark — APPROVED** [review]

**Comment by ghost** [comment]
i ded tag  pls

also BOOOO giving secborgs limited lasers BOOOO

**Comment by ShadowDeath6** [comment]
They still have infinite lasers

**Comment by ShadowDeath6** [comment]
It's been 24 hours

wake me up

**Pull request merged**